etc/kayobe/kolla.yml

```diff
@@ -431,6 +445,12 @@ kolla_build_customizations_common:
     - /additions/*
   ironic_inspector_pip_packages_append:
     - /additions/*
+  neutron_server_packages_append:
```

Comment:
Shouldn't that be added upstream?

Reply:
I'll respond to myself - since upstream is sort of dead, then it's hard to add it to Kolla ;-)